conf/messages

```diff
@@ -65,6 +65,16 @@ common.aboutGeneralIncreases.agent = the ’About general increases in benefits
 
 common.employerFinancedRetirementScheme.caption = Employer-financed retirement benefits schemes for 6 April {0} to 5 April {1}
 
+################# Common lifetime allowance #################
+
+common.lifetimeAllowance.Pension.checkWithProvider.Paragraph.individual = Check with your pension providers if you’re unsure.
+common.lifetimeAllowance.Pension.checkWithProvider.Paragraph.agent = Your client can check with their pension provider if you’re unsure.
+common.total.taxPaid.title = Total tax paid
+common.totalAmount.beforeTax.title = Total amount before tax
+common.beforeTax.error.overMaximum = The amount of lifetime allowance must be less than 100,000,000,000
```

Comment:
If this is for lifetime allowance, shouldn't the key reflect his, as we have for `common.lifetimeAllowance`?

As well, please see [the latter comments in the ticket](https://jira.tools.tax.service.gov.uk/browse/SASS-2465) for a correction of these two "maximum" error messages.